src/components/views/rooms/MessageComposer.tsx

```diff
@@ -154,6 +182,52 @@ export class MessageComposer extends React.Component<IProps, IState> {
         SettingsStore.monitorSetting("feature_wysiwyg_composer", null);
     }
 
+    private get editorStateKey(): string {
+        let key = `mx_wysiwyg_state_${this.props.room.roomId}`;
```

Comment:
Given there's no code to clean up drafts from rooms which you are no longer in can we move it to IDB where we have less space pressure?

Reply:
@t3chguy IDB appears to be a bit slower for small values than localstorage and because it's async setting the initial content outside of the constructor makes the initialisation of the editor feel a bit slow/janky. And introduced a weird issue where the content is initial text is sometimes selected.

Maybe we could stick with localstorage for consistency between it and CIDER but then look to add the cleanup for both? Is it just clearing the data when we leave or are kicked from a room?

Any other ideas?

Reply:
It'd have to be a periodic scan comparing the drafts stored vs known rooms at a known good client state, as the client may not receive kicks/leaves depending on if the room was forgotten via another client before it syncs

Reply:
Added